CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+
+### Finding Existing Issues
+
+Before filing a new issue, please search our [open issues](https://github.com/nmshd/feedback/issues) to check if it already exists.
+
+If you do find an existing issue, please include your own feedback in the discussion. Do consider upvoting (👍 reaction) the original post, as this helps us prioritize popular issues in our backlog.
+
+### Writing a Good Bug Report
+
+Good bug reports make it easier for maintainers to verify and root cause the underlying problem. The better a bug report, the faster the problem will be resolved. Ideally, a bug report should contain the following information:
+
+- A high-level description of the problem.
+- A _minimal reproduction_, i.e. the smallest size of code/configuration required to reproduce the wrong behavior.
+- A description of the _expected behavior_, contrasted with the _actual behavior_ observed.
+- Information on the environment (where applicable): OS/distro, CPU arch, SDK version, etc.
+- Additional information, e.g. is it a regression from previous versions? are there any known workarounds?
+
+When ready to submit a bug report, please use the [Bug Report issue template](https://github.com/nmshd/feedback/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D+).
+
+#### Why are Minimal Reproductions Important?
+
+A reproduction lets maintainers verify the presence of a bug, and diagnose the issue using a debugger. A _minimal_ reproduction is the smallest possible console application demonstrating that bug. Minimal reproductions are generally preferable since they:
+
+1. Focus debugging efforts on a simple code snippet,
+2. Ensure that the problem is not caused by unrelated dependencies/configuration,
+3. Avoid the need to share production codebases.
+
+#### Are Minimal Reproductions Required?
+
+In certain cases, creating a minimal reproduction might not be practical (e.g. due to nondeterministic factors, external dependencies). In such cases you would be asked to provide as much information as possible, for example by sharing a memory dump of the failing application. If maintainers are unable to root cause the problem, they might still close the issue as not actionable. While not required, minimal reproductions are strongly encouraged and will significantly improve the chances of your issue being prioritized and fixed by the maintainers.
+
+#### How to Create a Minimal Reproduction
+
+The best way to create a minimal reproduction is gradually removing code and dependencies from a reproducing app, until the problem no longer occurs. A good minimal reproduction:
+
+- Excludes all unnecessary types, methods, code blocks, source files, nuget dependencies and project configurations.
+- Contains documentation or code comments illustrating expected vs actual behavior.
+- If possible, avoids performing any unneeded IO or system calls.
+
+## Contributing Changes
+
+Project maintainers will merge changes that improve the product significantly.
```

Comment:
Hier fehlen irgendwie noch infos bzgl. PRs, z.B. auch How to write a nice PR, etc?

Reply:
Ich hab das mal bewusst weggelassen. Können wir ja noch nachreichen. Mir war wichtig, dass wir erstmal schnell etwas haben.